platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInstallCommand.java

```diff
@@ -0,0 +1,231 @@
+        }
+    }
+
+    @Override
+    protected void doExecute(ApplicationService applicationService)
+            throws ApplicationServiceException {
+
+        if (!profileExists(applicationService.getInstallationProfiles(), profileName) && showDevProfiles) {
+            Map<String, List<String>> devProfile = getDevProfile(profileName);
+
+            startApps(applicationService, devProfile.get(DEV_PROFILE_START_APPS));
+
+            if (featuresService != null) {
+                uninstallInstallerModule();
+                installFeatures(devProfile.get(DEV_PROFILE_INSTALL_FEATURES));
+                uninstallFeatures(devProfile.get(DEV_PROFILE_UNINSTALL_FEATURES));
+            }
+
+            stopBundles(devProfile.get(DEV_PROFILE_STOP_BUNDLES));
+
+            console.println(ansi().fgBrightGreen().a("Installation complete.").reset());
+        } else {
+            Feature profile = getProfile(applicationService, profileName);
+            List<String> profileApps = profile.getDependencies().stream()
+                    .map(Dependency::getName)
+                    .collect(Collectors.toList());
+
+            startApps(applicationService, profileApps);
+            uninstallInstallerModule();
+        }
+    }
+
+    private void startApps(ApplicationService applicationService, List<String> startupApps)
+            throws ApplicationServiceException {
+        if (startupApps != null) {
+            printSectionHeading("Starting Applications");
+            startupApps.forEach(application -> {
+                printItemStatusPending("Starting: ", application);
+                executeAsSystem(() -> {
+                    applicationService.startApplication(application);
+                    return true;
+                });
+                if (applicationService.isApplicationStarted(applicationService.getApplication(application))) {
+                    printItemStatusSuccess("Started: ", application);
+                } else {
+                    printItemStatusFailure("Start Failed: ", application);
+                }
+            });
+        }
+    }
+
+    private void uninstallInstallerModule() {
+        try {
+            if (!featuresService.isInstalled(featuresService.getFeature("admin-post-install-modules"))) {
+                featuresService.installFeature("admin-post-install-modules",
+                        EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+            }
+            if (featuresService.isInstalled(featuresService.getFeature("admin-modules-installer"))) {
+                console.println("Uninstalling installer");
+                featuresService.uninstallFeature("admin-modules-installer",
+                        EnumSet.of(FeaturesService.Option.NoAutoRefreshBundles));
+            }
+        } catch (Exception e) {
+            console.println(ansi().fgRed()
+                    .a("An error occurred while trying to perform post-install operations").reset());
```

Comment:
This message re-enforce the idea that we should do this task after having done all the others instead of before as done before.